### PR TITLE
Support comments

### PR DESCRIPTION
**Pull request by GerardPaligot**
* Parse comments from OpenFeedback firestore instance
* Create adapted UI for M3 implementation
* Create and edit new comment
* Up/Down votes a comment

### CODE REVIEW

File: openfeedback/src/main/java/io/openfeedback/android/mappers/FirestoreToModelMappers.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+internal fun Map<String, *>.convertToModel(
+    id: String,
+    voteItemId: String
+): Comment = Comment(
+    id = id,
+    voteItemId = voteItemId,
+    text = this["text"] as String,
+    plus = this["plus"] as Long,
+    createdAt = this["createdAt"] as Timestamp,
```
Comment (HugoGresse) [inline openfeedback/src/main/java/io/openfeedback/android/mappers/FirestoreToModelMappers.kt:14]: maybe FieldValue.serverTimestamp() ?

File: openfeedback-m3/src/main/kotlin/io/openfeedback/android/m3/OpenFeedbackLayout.kt [hunk 2]
```diff
@@ -54,10 +54,24 @@ fun OpenFeedback(
...
+                    )
+                },
+                commentInput = {
+                    CommentInput(
+                        value = session.commentValue,
+                        onValueChange = viewModel::valueChangedComment,
+                        onSubmit = viewModel::submitComment,
+                        modifier = Modifier.fillMaxWidth()
+                    )
+                },
```
Comment (martinbonnin) [inline openfeedback-m3/src/main/kotlin/io/openfeedback/android/m3/OpenFeedbackLayout.kt:70]: Should this be hidden if `commentVoteItemId` is null?

Comment (GerardPaligot) [reply openfeedback-m3/src/main/kotlin/io/openfeedback/android/m3/OpenFeedbackLayout.kt:70]: Good idea!

File: openfeedback/src/main/java/io/openfeedback/android/mappers/FirestoreToModelMappers.kt (hunk 1 above)
Comment (GerardPaligot) [reply openfeedback/src/main/java/io/openfeedback/android/mappers/FirestoreToModelMappers.kt:14]: Aaaah! There is a topic about all dates inside this project which use `java.util.Date` but I didn't know that we can use this too to initialize dates everywhere. I'll make the change!

**Pull request merged**